### DISCUSSION TITLE
`pairplot` from Seaborn

### DISCUSSION THREAD

**Discussion by ChristopherDavisUCI** [vega/vega-lite#8626] [Ideas]
Hi!  I often find myself wishing there were a concise way in Altair to reproduce the [pairplot function](https://seaborn.pydata.org/generated/seaborn.pairplot.html) in Seaborn (or the similar [scatterplot matrix](https://plotly.com/python/splom/) in Plotly Express).

@joelostblom and I were [talking about](https://github.com/altair-viz/altair/discussions/2789) adding something like this in Altair.  It would just be some shortcut I assume using the `repeat` operator.  We would probably also add some additional Exploratory Data Analysis shortcut functions, along the lines of what @joelostblom has already written for [Altair Ally](https://joelostblom.github.io/altair_ally/examples.html).

Before we started on that (it's not a big priority), I wanted to check to see if there was any interest in having something like that on the Vega-Lite side.  I don't know if these kind of shortcuts are in the spirit of Vega-Lite or not.

Thanks either way for any feedback!

**Comment by joelostblom** [comment]
I asked Dominik and Jeff about this and they mentioned that it is likely out of scope of Vega-Lite and would make more sense to add directly in Altair, so let's continue this discussion in https://github.com/altair-viz/altair/discussions/2789 instead
